src/content/wiki/tutorials/schematics.mdoc

```diff
@@ -319,8 +319,27 @@ So, for example, the path would be `blueprints/wildwest/fundamentals/builder1` f
 
 Because these have a fixed name, you can't have more than one per style and type.
 {% /version %}
+## Supplycamp/ship Content Requirements
+We have a set of rules a supplycamp/ship needs to adhere to as their placement is free in the world:
 
-You should always apply a `groundlevel` tag using the {% item name="structurize/tag_tool" /%} when making a supply camp/ship. The rules for guessing where the ground level is without the tag change depending whether they're placed by the {% item name="structurize/build_tool" /%} or the {% item_page name="supply_camp_and_ship" /%}, for legacy reasons.
+Size: Maximum 32x32 blocks
+
+Materials:
+- Place to two stacks of 2-3 main materials used in lower levels, e.g. Cobblestone, Oak logs and diorite.
+- 12 White wool for beds
+- Other low level materials are fine, keep their amount in moderation <~32, use DO framed/cut blocks else
```

Comment:
DO -> page link to Domum Ornamentum and probably write out the mod name in full, no shortcuts in the wiki since dummies read this and don't know what DO means.

Also replace <~32 with "less than 32", it's a wiki, not code